### ISSUE TITLE
Only a blank window is shown

### ISSUE THREAD

**Issue by yhylord** [equalsraf/neovim-qt#130]
I'm using Arch Linux with Linux 4.5.0-1. I've installed it by the package in AUR. The compiler is GCC 6.1.1 and the Qt version is 5.6.0-1.
I tried running `nvim-qt` under command prompt, but it complains nothing.
![screenshot_20160526_220449](https://cloud.githubusercontent.com/assets/5259071/15577171/f4ffbdb8-2320-11e6-8251-ce3d5943d4ee.png)

**Comment by equalsraf** [comment]
@yhylord maybe its stuck on neovim initialization, what happens if you run `nvim-qt -- -u NONE` ?

**Comment by yhylord** [comment]
It's the same. No output from the command line or anything displayed.

**Comment by equalsraf** [comment]
Maybe its the same as https://github.com/equalsraf/neovim-qt/issues/120#issuecomment-214545358

**Comment by yhylord** [comment]
Exactly! After installing Inconsolata everything turns fine. Thanks!

**Comment by equalsraf** [comment]
Leaving this open because I have no way to reproduce ATM, need a way to reliably load a fallback monospace font in Qt.

**Comment by htfy96** [comment]
Confirmed on my Arch Linux. Since I am using my own font configuration, installing Inconsolata doesn't work to me.

```
fc-match monospace
wqy-zenhei.ttc: "文泉驿等宽正黑" "Regular" // this is a Chinese font
```

**Comment by kiryl** [comment]
It seems the problem triggers when font has ligatures. I use PragmataPro as monospace alias and it leads to blank screen. Changing it to PragmataProMono (which is the same font, but without ligatures) it works fine.

It also worth noting that setting font in ginit.vim doesn't help as it loaded too late when we already stuck at blank screen.

**Comment by lilydjwg** [comment]
I've added this to my fontconfig file to avoid the problem:

```xml
  <match target="pattern">
    <test name="prgname" compare="eq">
      <string>nvim-qt</string>
    </test>
    <edit name="family" mode="prepend" binding="strong">
      <string>DejaVu Sans Mono</string>
    </edit>
  </match>
```

neovim-qt doesn't like Emoji One Color.

**Comment by sliceofcode** [comment]
Okay, so it looks like non-monospace fonts cause it to happen.

My fontconfig configuration forced `monospace` to resolve to Iosevka:
```
[ch:~]$ fc-match monospace
iosevka-regular.ttf: "Iosevka" "Regular"
```

Note that while Iosevka is a programming font and looks like its monospace, "Iosevka" is the not actually strictly monospace. After installing "Iosevka Term", which is the strictly monospace variant of Iosevka and aliasing it to monospace, `neovim-qt` launches.

**TL;DR**: `monospace` resolved to a font that wasn't strictly monospace. Nvim-qt then refuses to launch.

[See the relevant change in fontconfig configuration in my dotfiles here.](https://github.com/sliceofcode/dotfiles/commit/d2bacb019292444c1bbc719b523bf8e23cecc59b#diff-97fe5b720eb8b88e81ffd16e285ac011R8)

**Comment by equalsraf** [comment]
@sliceofcode thanks for the check, starting up #242 for this

**Comment by equalsraf** [comment]
This should be fixed in in master

**Comment by justinmk** [comment]
@equalsraf Any plans to publish a release?

**Comment by equalsraf** [comment]
@justinmk long overdue, but also waiting to see if #248 is a duplicate of this one or not
